Highway.cpp

```diff
@@ -13,22 +18,44 @@ void Highway::changeSpeed(int newSpeed)
 
 void Highway::addVehicleInternal(Vehicle* v)
 {
-    assert(false);
+    //assert(false);
 
     /*
     depending on the derived type, call the member function that doesn't evade the cops. 
     */
+    Car* car = dynamic_cast<Car*>(v);
+    if(car) { car->closeWindows(); }
+
+    Motorcycle* bike = dynamic_cast<Motorcycle*>(v);
+    if(bike) {  bike->lanesplitAndRace(200); }
+
+    SemiTruck* semi = dynamic_cast<SemiTruck*>(v);
+    if(semi) {  }
+
+    if(v) { v->setSpeed(65); }
```

Comment:
you are trying to cast `v` after a cast succeeds
- [ ] use `if/else if` 

you are missing a SemiTruck member function that doesn't evade the cops. 
- [ ] add and call a SemiTruck member function.  Check other students' projects for inspiration.